src/main.py

```diff
@@ -64,6 +66,38 @@ async def r_get(req, name):
     return response.json(valid_web_ids)
 
 
+@app.route('/candidates', methods=['GET'])
+async def get_handler(req):
+    sparql_db_url = environ.get('SPARQL_DBNAME')
```

Comment:
I'm being really pedantic here, but maybe rename the environment variable to `SPARQL_HOST`/`SPARQL_URL` or something like that, as it's the whole URL and not just the database name

Reply:
I'm on it!